Cesium3DTiles/src/CreditSystem.cpp

```diff
@@ -19,13 +23,15 @@ Credit CreditSystem::createCredit(const std::string& html) {
 }
 
 const std::string& CreditSystem::getHtml(Credit credit) const {
+  std::lock_guard<std::mutex> lock(_mutex);
   if (credit.id < _credits.size()) {
     return _credits[credit.id].html;
   }
   return INVALID_CREDIT_MESSAGE;
 }
 
 void CreditSystem::addCreditToFrame(Credit credit) {
+  std::lock_guard<std::mutex> lock(_mutex);
```

Comment:
This lock is likely to be very expensive, as it can happen hundreds or even thousands of times per frame. I have to think about how we can avoid it...